pcap-linux.c

```diff
@@ -1633,10 +1633,10 @@ pcap_activate_linux(pcap_t *handle)
 	/*
 	 * If we're in promiscuous mode, then we probably want
 	 * to see when the interface drops packets too, so get an
-	 * initial count from /proc/net/dev
+	 * initial count from /sys/class/net
```

Comment:
Perhaps that should be "initial count from /sys/class/net/statistics/{if_name}/rx_{missed,fifo}_errors", as per another comment.